apis/config/v1/types.go

```diff
@@ -149,6 +149,8 @@ type NodeResourceTopologyMatchArgs struct {
 	ScoringStrategy *ScoringStrategy `json:"scoringStrategy,omitempty"`
 	// If > 0, enables the caching facilities of the reserve plugin - which must be enabled
 	CacheResyncPeriodSeconds *int64 `json:"cacheResyncPeriodSeconds,omitempty"`
+	// if set to true, exclude node from scheduling if there are any reserved pods for given node
+	DiscardReservedNodes bool
```

Comment:
missing JSON annotations. Please make sure the field is fully optional. Do you need to distinguish between "unspecified" and "zero"?

Reply:
I don't need to use `*bool` I can rely on default value of boolean which is `false` 😄

Reply:
fair enough!